schemas/compiler_pass_v1.json

```diff
@@ -834,13 +834,13 @@
         "timeout": {
           "type": "integer",
           "minimum": 0,
-          "description": "Timeout for vf2 search, in seconds."
+          "description": "Timeout for monomorphism search, in milliseconds."
```

Comment:
I assume this was a bug, we aren't changing it?

Reply:
Yeah the json was incorrect previously